backend/qemu.pm

```diff
@@ -710,6 +710,7 @@ sub start_qemu {
     my $arch_supports_boot_order = $vars->{UEFI} ? 0 : 1;    # UEFI/OVMF supports ",bootindex=N", but not "-boot order=X"
     my $use_usb_kbd;
     my $arch = $vars->{ARCH} // '';
+    $arch = 'arm' if ($vars->{ARCH} =~ /armv6|armv7/);
```

Comment:
The test likely fails because `$vars->{ARCH}` might be uninitialized here producing a warning. This likely should have been just `$arch`.

Reply:
Yes, I noticed it afterwards, I just pushed a fix.